niworkflows/viz/plot.py

```diff
@@ -0,0 +1,176 @@
+    """ Plot contours """
+    input_spec = PlotContoursInputSpec
+    output_spec = PlotContoursOutputSpec
+
+    def __init__(self, **inputs):
+        self._results = {}
+        super(PlotContours, self).__init__(**inputs)
+
+    def _list_outputs(self):
+        return self._results
+
+    def _run_interface(self, runtime):
+        out_file = None
+        if isdefined(self.inputs.out_file):
+            out_file = self.inputs.out_file
+
+        fname, _ = split_ext(self.inputs.in_file, out_file)
+        out_file = op.abspath('plot_' + fname + '_contours.svg')
+        self._results['out_file'] = out_file
+
+        plot_segmentation(
+            self.inputs.in_file,
+            self.inputs.in_contours,
+            out_file=out_file,
+            cut_coords=self.inputs.cut_coords,
+            display_mode=self.inputs.display_mode,
+            levels=self.inputs.levels,
+            colors=self.inputs.colors,
+            saturate=self.inputs.saturate)
+
+        return runtime
+
+class PlotMosaicInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True,
+                   desc='File to be plotted')
+    in_mask = File(exists=True, desc='Overlay mask')
+    title = traits.Str('Volume', usedefault=True,
+                       desc='modality name to be prepended')
+    subject = traits.Str(desc='Subject id')
+    metadata = traits.List(traits.Str, desc='additional metadata')
+    figsize = traits.Tuple(
+        (11.69, 8.27), traits.Float, traits.Float, usedefault=True,
+        desc='Figure size')
+    dpi = traits.Int(300, usedefault=True, desc='Desired DPI of figure')
+    out_file = File('mosaic.pdf', usedefault=True, desc='output file name')
+
+
+class PlotMosaicOutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='output pdf file')
+
+
+class PlotMosaic(BaseInterface):
+
+    """
+    Plots slices of a 3D volume into a pdf file
+    """
+    input_spec = PlotMosaicInputSpec
+    output_spec = PlotMosaicOutputSpec
+
+    def _run_interface(self, runtime):
+        mask = None
+        if isdefined(self.inputs.in_mask):
+            mask = self.inputs.in_mask
+
+        title = self.inputs.title
+        if isdefined(self.inputs.subject):
+            title += ', subject {}'.format(self.inputs.subject)
+
+        if isdefined(self.inputs.metadata):
+            title += ' (' + '_'.join(self.inputs.metadata) + ')'
+
+        figsize = None
+        if isdefined(self.inputs.figsize):
+            figsize = self.inputs.figsize
+
+        fig = plot_mosaic(
+            self.inputs.in_file,
+            title=title,
+            overlay_mask=mask,
+            figsize=figsize)
+
+        fig.savefig(self.inputs.out_file, dpi=self.inputs.dpi)
+        fig.clf()
+        return runtime
+
+    def _list_outputs(self):
+        outputs = self.output_spec().get()
+        outputs['out_file'] = op.abspath(self.inputs.out_file)
+        return outputs
+
+
+class PlotFDInputSpec(BaseInterfaceInputSpec):
```

Comment:
The ```PlotFD``` has been removed from mriqc, because the code now lives in nipype. This interface should go away.